src/main/java/com/exasol/adapter/installer/dialect/PostgresDialectProfile.java

```diff
@@ -44,4 +44,14 @@ public String getDefaultPort() {
     protected String getDefaultDriverName() {
         return "postgresql.jar";
     }
+
+    @Override
+    public String getConnectionString() {
+        final String connectionString = getDriverPrefix() + "//" + getHost() + ":" + getPort() + "/";
+        if (hasAdditionalConnectionProperties()) {
+            return connectionString + getAdditionalConnectionProperties();
```

Comment:
ah so they differ in the missing slash here?
Maybe you could configure that by a var like `additionalProperySeperator`?

Reply:
They are quite different, Athena doesn't have a port for example, and so on. Avoiding duplication here can lead to mistakes in logic, that's why I had to separate them.